### ISSUE TITLE
Language does not exist *

### ISSUE THREAD

**Issue by HyunSangHan** [11ty/eleventy#730]
**Describe the bug**
I was trying to build [docs of Mochajs](https://mochajs.org) for dev, and I found the error messages which are `Language does not exist sh` and `Language does not exist text`. It is only shown when I build docs by `eleventy`.

**To Reproduce**
1. Clone [Mochajs](https://github.com/mochajs/mocha)
2. `npm install`
3. `npm start docs` => It will make you run `eleventy`

**Expected behavior**
The error messages will be shown as below:
```
Language does not exist sh
Language does not exist sh
Language does not exist sh
Language does not exist sh
Language does not exist sh
Language does not exist sh
Language does not exist sh
Language does not exist text
Language does not exist sh
Language does not exist sh
Language does not exist sh
Language does not exist sh
Language does not exist sh
Language does not exist sh
```
**Screenshots**

```console
Hyunsangs-MacBookAir:mocha temp$ npm start docs

> mocha@6.2.0 start /Users/temp/Downloads/exercise/mocha
> nps "docs"

nps is executing `docs` : nps docs.prebuild && nps docs.api && eleventy && nps docs.linkcheck && nps docs.postbuild
nps is executing `docs.prebuild` : rimraf docs/_dist docs/_site && nps docs.preprocess
nps is executing `docs.preprocess` : md-magic --config ./scripts/markdown-magic.config.js --path docs/index.md
Starting markdown-magic docs/index.md
✔ docs/index.md Updated
 Transforms run
  ⁕ usage:executable=bin/mocha
  ⁕ toc:maxdepth=2&bullets=-
  ⁕ manifest:template=[gitter]: ${gitter}

Files processed. markdown-magic Finished! ⊂◉‿◉つ
nps is executing `docs.api` : nps docs.preprocess.api && jsdoc -c jsdoc.conf.json
nps is executing `docs.preprocess.api` : md-magic --config ./scripts/markdown-magic.config.js --path "docs/api-tutorials/*.md"
Starting markdown-magic docs/api-tutorials/*.md
✔ docs/api-tutorials/custom-reporter.md Updated
 Transforms run
  ⁕ file:src=../../test/integration/fixtures/simple-reporter.js&header=// my-reporter.js

Files processed. markdown-magic Finished! ⊂◉‿◉つ
Parsing /Users/temp/Downloads/exercise/mocha/lib/browser/growl.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/browser/progress.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/browser/tty.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/cli/cli.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/cli/collect-files.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/cli/commands.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/cli/config.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/cli/index.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/cli/init.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/cli/node-flags.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/cli/one-and-dones.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/cli/options.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/cli/run-helpers.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/cli/run-option-metadata.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/cli/run.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/cli/watch-run.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/context.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/errors.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/growl.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/hook.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/interfaces/bdd.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/interfaces/common.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/interfaces/exports.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/interfaces/index.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/interfaces/qunit.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/interfaces/tdd.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/mocha.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/pending.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/reporters/base.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/reporters/doc.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/reporters/dot.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/reporters/html.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/reporters/index.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/reporters/json-stream.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/reporters/json.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/reporters/landing.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/reporters/list.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/reporters/markdown.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/reporters/min.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/reporters/nyan.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/reporters/progress.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/reporters/spec.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/reporters/tap.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/reporters/xunit.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/runnable.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/runner.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/stats-collector.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/suite.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/test.js ...
Parsing /Users/temp/Downloads/exercise/mocha/lib/utils.js ...
Generating output files...
Finished running in 2.07 seconds.
Language does not exist sh
Language does not exist sh
Language does not exist sh
Language does not exist sh
Language does not exist sh
Language does not exist sh
Language does not exist sh
Language does not exist text
Language does not exist sh
Language does not exist sh
Language does not exist sh
Language does not exist sh
Language does not exist sh
Language does not exist sh

```
**Environment:**
 - OS and Version: Mac
 - Eleventy Version: 0.8.3

**Additional context**
Even if there are some error messgaes, I think there might be no problem to build docs. I just want to know why it is happend and how I can fix it. :)

**Comment by Ryuno-Ki** [comment]
My bet: You're using code fences (triple backticks) with `sh` for highlight.
Eleventy is using https://github.com/11ty/eleventy-plugin-syntaxhighlight for syntax highlight, which is utilising Prism.
The supported languages are https://prismjs.com/#languages-list

TL;DR: Use `shell` or `bash` instead of `sh`. Use nothing for plain/text.

**Comment by HyunSangHan** [comment]
@Ryuno-Ki 
You are right! Thank you for your reply.
